### PR TITLE
[Bagatji drink recipe] There's a Snake in my Bar

### PR DESCRIPTION
**Pull request by NotRanged**
## About The Pull Request

Made on request of Xyverious.

Adds ~~Bagatji~~ Balakit Wine (5 wine, 2 toxin, 3 mindbreaker). On consumption, it does toxins, staminaloss and makes you hallucinate for a while. Characters with naga bodies are immune to these effects.

## Why It's Good For The Game

Snek wine!

## Changelog
:cl:
add: Added drink recipe 'Balakit Wine'. Made with 5u wine, 2u toxin, 3u mindbreaker. Result is 5u. Consumption by non-naga characters is not recommended.
/:cl:

### CODE REVIEW

**Comment by CameronWoof** [comment]
Xyverious is cool but adding custom stuff for a race you made up is pretty lame. Also, 50 tox damage per sip is pretty silly.

**Comment by NotRanged** [comment]
50 toxin per sip spread out over more than 2 minutes. 0.4 tox per second would take any minutes to kill, if you can't find antitox in that time it's on you tbh

**Comment by CameronWoof** [comment]
I just generally feel like this is content with such a narrow audience that it feels bad on principle. It feels like codifying a sort of special snake club by giving them content that only they can safely enjoy. The fact that it's named after someone's OC race makes it feel like that special lavaland dragon thing all over again.

**Comment by NotRanged** [comment]
Numbers changed. 5u now does 39 toxin damage over a 25 second period.

**Comment by Echopsychotic** [comment]
just be a slime and enjoy the drink

**Comment by KathrinBailey** [comment]
image

File: modular_skyrat/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -91,4 +91,50 @@
...
+	glass_icon = 'modular_skyrat/icons/obj/drinks.dmi'
+	glass_icon_state = "bagatjiwineglass"
+	glass_name = "Balikit Wine"
+	glass_desc = "They put it in a decanter."
+	value = REAGENT_VALUE_RARE
+
+/datum/reagent/consumable/ethanol/balikitwine/on_mob_life(mob/living/carbon/C)
+	var/mob/living/carbon/human/H = C
+	if(istype(H) && H.physiology.footstep_type != FOOTSTEP_MOB_CRAWL) // Bit of a hacky way to check if the consumer is a snake taur. Only snektaurs have this though.
+		H.adjustToxLoss(3*REM) // Bagatji (and all snake taurs) take no ill effects.
```
Comment (KathrinBailey) [inline modular_skyrat/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:114]: Everything but this is okay. It's currently 39 toxin damage per single sip, given the not super hard to obtain crafting recipe it probably should be lowered just a tad.
Lowering this to 2.5 will deal 32.5 toxin damage per sip.
Lowering this to 2 will deal 26 toxin damage per sip.

Change to either 2 or 2.5 and ping me at Floof Ball#0798 once done and once I'm able I'll merge it. I can quickly do it for you if you also ping me to say it's okay.

**Review by KathrinBailey — CHANGES_REQUESTED** [review]
You just need to lower the toxin damage a tiny bit.

**Review by KathrinBailey — APPROVED** [review]

**Pull request merged**